Spigot-Server-Patches/0679-Add-Multiple-Entries-to-Scoreboard-Team.patch

```diff
@@ -0,0 +1,109 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Tom <cryptite@gmail.com>
+Date: Fri, 26 Feb 2021 17:02:17 -0600
+Subject: [PATCH] Add Multiple Entries to Scoreboard Team
+
+
+diff --git a/src/main/java/net/minecraft/server/Scoreboard.java b/src/main/java/net/minecraft/server/Scoreboard.java
+index 2ab11597adb1c713011103cfc53011f93bbb0336..ef1897c11e958346003279e7a9a08fd79700aad9 100644
+--- a/src/main/java/net/minecraft/server/Scoreboard.java
++++ b/src/main/java/net/minecraft/server/Scoreboard.java
+@@ -50,7 +50,7 @@ public class Scoreboard {
+ 
+     public final void getObjectivesForCriteria(IScoreboardCriteria iscoreboardcriteria, String s, Consumer<ScoreboardScore> consumer) {
+         ((List) this.objectivesByCriteria.getOrDefault(iscoreboardcriteria, Collections.emptyList())).forEach((scoreboardobjective) -> {
+-            consumer.accept(this.getPlayerScoreForObjective(s, scoreboardobjective));
++            consumer.accept(this.getPlayerScoreForObjective(s, (ScoreboardObjective) scoreboardobjective)); // Paper
+         });
+     }
+ 
+@@ -231,6 +231,28 @@ public class Scoreboard {
+         }
+     }
+ 
++    // Paper start
++    public boolean addPlayersToTeam(Collection<String> entries, ScoreboardTeam scoreboardteam) {
++        boolean added = false;
++        for (String entry : entries) {
++            if (entry.length() > 40) {
```

Comment:
This logic here just feels like this will potentially cause a desync if there is a name in the middle which is bad?